internal/textutil/diff.go

```diff
@@ -0,0 +1,74 @@
+package textutil
```

Comment:
I presume that this isn't from the Go repo? I think we should start labelling what packages mirror the Go repo, and what packages are ours.

Reply:
No, it's not. And good point. I'll add a quick package comment along with a note on how it came to life.